### PR TITLE
[stable/elasticsearch-exporter] ServiceMonitor: set namespace to monitor + Add OWNER and add myself to owners.

### PR DESCRIPTION
**Pull request by desaintmartin**
#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes #

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://www.helm.sh/blog/helm-dco/index.html) signed
- [x] Chart Version bumped

### CODE REVIEW

**Review by scottrigby — APPROVED** [review]

File: stable/elasticsearch-exporter/.helmignore [hunk 1]
```diff
@@ -0,0 +1,3 @@
+.git
```
Comment (scottrigby) [inline stable/elasticsearch-exporter/.helmignore]: Do you want to go ahead and add the other common patterns from `helm create`?

Comment (desaintmartin) [reply stable/elasticsearch-exporter/.helmignore]: Done!

**Pull request merged**